helper/reports.go

```diff
@@ -0,0 +1,76 @@
+package helper
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+)
+
+func ProcessDownloadReports(fileID int, alwaysApproved bool, generate string, output string) (bool, error) {
+	var resultID int
+
+	fmt.Println("Warning: This process will download report file to system.")
+	if !alwaysApproved {
+		fmt.Println("Please pass `--always-approved` to approve all the reports")
+		return false, errors.New("Please pass `--always-approved` to approve all the reports")
+	}
+
+	ctx := context.Background()
+	client := getClient()
+
+	if generate == "yes" {
```

Comment:
If the value of generate flag can only be `yes` or `no`, why don't we make it a boolean flag. The absence of flag will mean `false` as we are doing with `always-approved` flag?

Reply:
Sure, we changing it.